.github/workflows/sub_gradle_test_and_build.yml

```diff
@@ -134,7 +130,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        language: [ 'java', 'python' ]
+        language: [ 'java' ]
```

Comment:
Can we try enabling Kotlin support too? `java` can be replaced `java-kotlin`. https://codeql.github.com/docs/codeql-language-guides/codeql-for-java/